src/appleseed-max-impl/appleseeddisneymtl/appleseeddisneymtl.cpp

```diff
@@ -591,6 +600,8 @@ Interval AppleseedDisneyMtl::Validity(TimeValue t)
 ParamDlg* AppleseedDisneyMtl::CreateParamDlg(HWND hwMtlEdit, IMtlParams* imp)
 {
     ParamDlg* param_dialog = g_appleseed_disneymtl_classdesc.CreateParamDlgs(hwMtlEdit, imp, this);
+    if (m_pblock != nullptr)
```

Comment:
Can `m_pblock` be null?

Reply:
I don't know honestly. I can put assert and try to do something with materials and see if it crashes.